processor/transformprocessor/config_test.go

```diff
@@ -44,6 +44,7 @@ func TestLoadingConfig(t *testing.T) {
 			Queries: []string{
 				`set(name, "bear") where attributes["http.path"] == "/animal"`,
 				`keep_keys(attributes, "http.method", "http.path")`,
+				`join("|", attributes["service.name"], attributes["service.name"], attributes["k8s.namespace.name"])`,
```

Comment:
Hi @JaredTan95 - thanks for this function implementation. Instead of `join`, what do you think about 

`concat(attributes["service.name"], "|", attributes["service.name"], "|", attributes["k8s.namespace.name"])`

? It is a bit more tedious for the join case but is otherwise much more flexible and still maybe satisfies the join case just fine. I think that either way we do need a concat function so maybe we can start with it.

Reply:
make sense, I will update it.

Reply:
If it's OK, I'd like to do a PR with concat first, and add join later if there is more demand. concat can satisfy the join case and others so is more important to add first

Reply:
ok